src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    @Override
+    public List<FruitTransaction> convertListToObjects(List<String> list) {
+        List<FruitTransaction> listOfTransactions = new ArrayList<>();
+        for (int i = 1; i < list.size(); i++) {
```

Comment:
Consider adding a check to ensure that the input list has a valid CSV format and contains the expected number of columns. This will help to avoid `StringIndexOutOfBoundsException` if the input list does not match the expected format.